apps/bmd/src/AppRoot.tsx

```diff
@@ -727,12 +727,20 @@ const AppRoot: React.FC<Props> = ({
           const isValid =
             cardData.h === optionalElectionDefinition?.electionHash
 
-          /* istanbul ignore next */
-          const possibleCardTally: Optional<CardTally> =
+          let possibleCardTally: Optional<CardTally> =
             isValid && !!longValueExists
               ? ((await card.readLongObject()) as Optional<CardTally>)
               : undefined
 
+          // Handle a possible invalid object in the card long value
+          if (
+            possibleCardTally?.metadata === undefined ||
+            possibleCardTally?.tally === undefined ||
+            possibleCardTally?.totalBallotsPrinted === undefined
+          ) {
+            possibleCardTally = undefined
+          }
```

Comment:
Would be great to use a `zod` schema to validate this. Wouldn't say it's required for this PR, but I think it's a habit we should get into.

Reply:
Will followup looking at zod in a future PR